.vscode/settings.json

```diff
@@ -14,11 +14,11 @@
         "source.organizeImports": true
     },
     "yaml.schemas": {
```

Comment:
I think we can remove "yaml.schemas" from settings.json, and put the "# yaml-language-server:" line at the start of each yaml file instead